NebulaWorld/SimulatedWorld.cs

```diff
@@ -75,15 +79,14 @@ public static void UpdatePlayerColor(ushort playerId, Float3 color)
 
         public static void UpdatePlayerColor(Transform transform, Float3 color)
         {
-            // Apply new color to each part of the mecha
-            Renderer[] componentsInChildren = transform.gameObject.GetComponentsInChildren<Renderer>(includeInactive: true);
+            Log.Info($"Changing color of {transform.gameObject.name} to {color}");
 
+            // Apply new color to each part of the mecha
+            Renderer[] componentsInChildren = transform.gameObject.GetComponentsInChildren<Renderer>(includeInactive: false);
```

Comment:
Is the change to `includeInactive: false` really something that you wanted ?

Reply:
Not necessarily, it was just in case something was disabled, but it seems it never happens, so ill remove it.